### ISSUE TITLE
Payout Method Incorrectly Set for Balance Transfer Expenses

### ISSUE THREAD

**Issue by SudharakaP** [opencollective/opencollective#7457]
**Describe the bug**

It seems that if one creates a balance transfer expense for example from a collective to a project or event; then the expense is created as an invite and there will be no payout method. A good example of this is, https://opencollective.com/missingpiece/expenses/207523. 

Here as you can see, it's created as an invite. However there's no person in particular being invited in this expense as the payee is an event. Also the payout method is not set. 

**To Reproduce**
1. Go to any collective with an event
2. Submit an expense from the collective to it's event

**Expected behavior**

With this kinda expense, the payout method should auto set to balance transfer, and given that the purpose is a balance transfer only host or collective admins should be able to create these (hence should go to pending state directly).

**Comment by znarf** [comment]
> It seems that if one creates a balance transfer expense for example from a collective to a project or event; then the expense is created as an invite and there will be no payout method

An invite really? Can't reproduce with the steps provided.

> Also the payout method is not set.

I can't see that, it seems to be set.

> the payout method should auto set to balance transfer

I don't understand what that means, there is no such thing as a "balance transfer" payout method. The payout method to move money between Collective and Parent is "Collective Balance."

> and given that the purpose is a balance transfer only host or collective admins should be able to create these

How should the person posting the expense express that the purpose is a "balance transfer", there is no such concept in our expense system. I don't get this.

**Comment by SudharakaP** [comment]
Yes, it was an invite before, but now it's all paid and the payout method is filled to "Collective Balance" (and yes, I mistakenly put balance transfer, it should be collective balance). I will check with Wayne on how this was later paid and get back.

**Comment by SudharakaP** [comment]
@znarf : Here's an expense I created in Staging with this state; https://staging.opencollective.com/octobox/expenses/10924. Where as the host admin I was able to invite an expense to a project. 

Same can be reproduced for an event as well as I've done here; https://staging.opencollective.com/dlang/expenses/10925

The collective balance payout method was not set and in these cases the admin of the collective confused as to what to do. Although I believe there's some workaround and I am waiting for Wayne as they somehow managed to get it into approved state after letting me on this issue. I'll post when they get back. In any case, I believe this is an issue given ideally the payout method for these kinda expenses should be set (given it's always collective balance) without any interference with the collective admin. The collective admin should just be there to approve.

**Comment by znarf** [comment]
Thank you for additional information.

The title of the issue is still misleading, the provided steps are still not detailed enough to reproduce the issue.

**Comment by znarf** [comment]
Then, when accepting the invitation, it's not possible to move forward without picking a Payout Method. And the only Payout Method available is "Collective Balance.

<img width="854" alt="Screenshot 2024-06-27 at 13 50 08" src="https://github.com/opencollective/opencollective/assets/806/6bfa2536-96f5-4ee0-8cb0-0abc74bbbe1c">


<img width="847" alt="Screenshot 2024-06-27 at 13 50 45" src="https://github.com/opencollective/opencollective/assets/806/ae7f2b59-09e1-4a43-a236-7de3ffb85ba6">
